src/Executables/ExportCoordinates/ExportCoordinates.hpp

```diff
@@ -35,11 +37,13 @@
 #include "ParallelAlgorithms/EventsAndTriggers/Event.hpp"
 #include "ParallelAlgorithms/EventsAndTriggers/EventsAndTriggers.hpp"
 #include "ParallelAlgorithms/EventsAndTriggers/Tags.hpp"
+#include "ParallelAlgorithms/Initialization/Actions/AddComputeTags.hpp"
 #include "ParallelAlgorithms/Initialization/Actions/RemoveOptionsAndTerminatePhase.hpp"
 #include "Time/Actions/AdvanceTime.hpp"
 #include "Time/TimeSteppers/TimeStepper.hpp"
 #include "Time/Triggers/TimeTriggers.hpp"
 #include "Utilities/Blas.hpp"
+#include "Utilities/Functional.hpp"
```

Comment:
You're missing `#include "Domain/MinimumGridSpacing.hpp"`. Please squash right away.

Reply:
Thank you for catching this! @nilsdeppe Wanted to let you know that I think this new push dismissed your review